xbmc/guilib/GUIFontManager.cpp

```diff
@@ -49,6 +49,8 @@ using namespace ADDON;
 namespace
 {
 constexpr const char* XML_FONTCACHE_FILENAME = "fontcache.xml";
+constexpr int XML_FONTCACHE_VER_0 = 0; // Initial version
```

Comment:
Mind clarifying why and when the font cache should be rebuilt? This is probably something that should be forced on startup on new releases (first run)? Maybe it should be the application in charge of triggering the rebuild and not this component.

Reply:
thanks for point this out i think this is a full oversight i got confused things last days
now rereading the code i realized that since we add new extension the operation is done automatically anyway
this is therefore superfluous i will remove it